.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Hydra Server:**
+- OS and version: [e.g. NixOS 22.05.20211203.ee3794c]
+- Version of Hydra
+- Version of Nix Hydra is built against
+- Version of the Nix daemon 
```

Comment:
Any commands to get these infos?

Reply:
I can't think of an easy way to do any of these today!

Reply:
`$(systemctl show nix-daemon.service|grep ExecStart=|grep -Po "(?<=path=).*?(?= )") --version`

The other two are harder. Maybe hydra should just expose them through the API somehow. AFAIR even what's displayed in the web UI is a lie and just taken from the storepaths or someplace like that. In fact, my hydra says "Hydra 2021-08-11 (using nix-2.5pre20211206_d1aaa7e).", which is _definitely_ wrong.

Reply:
@samueldr What if I put a thing before this saying "It is fine to leave these fields empty if you don't know how to answer them." ?

Reply:
Just (optional) in the same line at the end?

Reply:
Sure, don't block on that. It was if possible :).

Reply:
:+1:  thanks!